CHANGELOG.md

```diff
@@ -0,0 +1,65 @@
+# Changelog
+
+Darts is still in an early development phase and we cannot always guarantee backwards compatibility. Changes suceptible to **break existing code** working on a previous release are marked with a "&#x1F534;". Sorry for the inconvenience!
+
+## [Unreleased](https://github.com/unit8co/darts/tree/develop)
+
+[Full Changelog](https://github.com/unit8co/darts/compare/0.3.0...develop)
+
+## [0.3.0](https://github.com/unit8co/darts/tree/0.3.0) (2020-10-01)
+
+[Full Changelog](https://github.com/unit8co/darts/compare/0.2.3...0.3.0)
+
+### For users of the library:
+**Added:**
+
+- Indexing on TimeSeries [\#150](https://github.com/unit8co/darts/pull/150)
```

Comment:
Maybe we need to specify that we're referring to column/component indexing, since indexing by time stamps was already present.